April TFE release small updates (#615)
hashicorp/terraform-docs-common: website/docs/cloud-docs/workspaces/settings/run-tasks.mdx

### Global Run Tasks

> When you create a new run task, you can choose to apply it globally to every workspace in an organization. This feature requires the **global-run-task** [entitlement][].

Reviewer: last sentences read a little funny - i assume the entitlement is part of a paid plan maybe? feature enabling an entitlement feels confusing, maybe im misunderstanding?

Author: Thank you for reminding me I wanted to look up how entitlements work! 

>Each organization has a set of entitlements that corresponds to its pricing tier. These entitlements determine which HCP Terraform features the organization can use.
>
> If an organization doesn't have the necessary entitlement to use a given feature, HCP Terraform returns a 404 error for API requests to any endpoints devoted to that feature.

So it sounds like you need entitlements to unlock specific features for an organization. Does this feel like a better way of putting this?
```suggestion
When you create a new run task, you can choose to apply it globally to every workspace in an organization. Your organization must have the `global-run-task` [entitlement][] to use global run tasks.
```
